Kitodo/src/main/webapp/WEB-INF/templates/includes/metadataEditor/gallery.xhtml

```diff
@@ -113,23 +121,29 @@
             <ui:fragment rendered="#{MetadataProcessor.viewMode eq 'preview' and MetadataProcessor.imageListExistent}">
                 <!-- OpenLayers Kitodo custom controls -->
                 <div id="olWrapper" style="overflow-x: auto; height: 100%">
-                    <div id="thumbnailStripe" style="width: 100px; display: inline-block; float: right">
-                        <ui:repeat value="#{MetadataProcessor.images}" var="image">
-                            <div>
-                                <p:commandLink update="imagePreviewForm, galleryHeadingWrapper" rendered="#{MetadataProcessor.isAccessGranted(image)}">
-                                    <!-- outputText is needed to remove whitespace produced by correct code indent -->
-                                    <h:outputText><img style="display: inline;" class="thumbnail #{MetadataProcessor.currentImage eq image ? 'active' : ''}" src="#{request.contextPath}#{MetadataProcessor.getThumbnail(image)}"/></h:outputText>
-                                    <f:setPropertyActionListener value="#{image}" target="#{MetadataProcessor.currentImage}"/>
-                                </p:commandLink>
+                    <div id="thumbnailStripe">
+                        <p:commandButton icon="fa fa-arrow-up fa-lg" id="scroll-up" styleClass="scroll-button scroll-up"/>
+                        <div id="thumbnailStripeScrollableContent">
+                            <div id="thumbnailWrapper">
+                                <ui:repeat value="#{MetadataProcessor.images}" var="image">
+                                    <div>
+                                        <p:commandLink update="imagePreviewForm, galleryHeadingWrapper" oncomplete="checkScrollPosition()" rendered="#{MetadataProcessor.isAccessGranted(image)}">
+                                            <!-- outputText is needed to remove whitespace produced by correct code indent -->
+                                            <h:outputText><img style="display: inline;" class="thumbnail #{MetadataProcessor.currentImage eq image ? 'active' : ''}" src="#{request.contextPath}#{MetadataProcessor.getThumbnail(image)}"/></h:outputText>
+                                            <f:setPropertyActionListener value="#{image}" target="#{MetadataProcessor.currentImage}"/>
+                                        </p:commandLink>
+                                    </div>
+                                    <!-- FIXME: primefaces does not find the image, need to prefix the path with 'window.location.host'! -->
+                                    <!--<p:graphicImage styleClass="thumbnail" name="#{image}" alt="#{image}" title="#{image}"/>-->
```

Comment:
Is this still a problem?

Reply:
This comment is outdated and should have been remove earlier.
I removed it now.